### PR TITLE
kujira assets

### PR DESCRIPTION
**Pull request by codehans**

### CODE REVIEW

**Review by dynalz — APPROVED** [review]

File: kujira/assetlist.json
Comment (sunnya97) [inline kujira/assetlist.json:377]: I think the name should be "Local Money" and that the description should be something longer

**Comment by JeremyParish69** [comment]
I helped draft out what I believe is the situation with LOCAL on Kujura:
https://github.com/cosmos/chain-registry/pull/3102/files#diff-1d9d2c8e24e586a0741ec2191ce697e973a65c99122e648f715c99be34eeca45
Although I don't know if the submitter has abandoned that PR.
Note how there is a trace for the asset and how we don't need to upload a duplicate copy of the logo image. I think we were just blocked on needing the IBC connection to Terra to be registered.

**Comment by codehans** [comment]
Yeah just adding this in so Pulsar can index it. Will leave it up to the Local DAO team to deal with the details. 

Not sure the context for the ibc channel here though. This is a factory token originating on the kujira network

**Comment by sunnya97** [comment]
I think they were also trying to register their legacy token from Terra Classic

**Comment by sunnya97** [comment]
But for this PR, should be fine

**Pull request merged**
